mysql_dump_script_docker.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+ 
+#date du jour
+DATE=`date +%y_%m_%d`
+ 
+#liste des dossier
+LISTEBDD=$( echo 'show databases' | mysql --user=root --password=wwitri2Dzhishme )
+ 
+#on boucle sur chaque dossier (for découpe automatiquement par l'espace)
+for SQL in $LISTEBDD
+ 
+do
+ 
+if [ $SQL == "piwik" ]; then
+ 
+#echo $SQL
+mysqldump --user=root --password=wwitri2Dzhishme $SQL | gzip > /home/$SQL"_mysql_docker"$DATE.sql.gz
```

Comment:
This script file should not have a password written in it, but should instead read a password from configuration file or command argument. Also the output path and the file name do not look correct

Reply:
I fully agree ... this is why I templated the mysqldump file